change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Any additional testing we should do when this is uptaken in SLE? If we look at the SystemLinkShared Chromatic build and fonts still work, does that seem sufficient to you?

Reply:
Yep, that would work. With the angular app in here building I'm pretty confident, but that would be a good double check 👍

Reply:
I'll validate in the renovate bubble-up

Reply:
Validated in [Skyline chromatic build](https://ni.visualstudio.com/DevCentral/_build/results?buildId=4635532&view=logs&j=5cb0c4d9-f36d-5fb3-bf2e-85320ef8bd68&t=8b524ba8-3d0d-5872-d8b6-3acaf409afeb)